documentation/docs/steps/abapEnvironmentRunAUnitTest.md

```diff
@@ -0,0 +1,144 @@
+* You can either provide the ABAP endpoint configuration to directly trigger an AUnit run on the ABAP system or optionally provide the Cloud Foundry parameters with your credentials to read a Service Key of a SAP BTP, ABAP environment system in Cloud Foundry that contains all the details of the ABAP endpoint to trigger an ATC run.
+* Regardless if you chose an ABAP endpoint directly or reading a Cloud Foundry Service Key, you have to provide the object set containing the objects you want to be checked in an AUnit run in a .yml or .yaml file. This file must be stored in the same folder as the Jenkinsfile defining the pipeline.
+* Make sure that the objects contained in the object set are present in the configured system in order to run the check. Please make sure that you have created or pulled the respective software components and/or Packages including the Test Classes and objects in the SAP BTP, ABAP environment system, that should be checked.
+
+## ${docGenParameters}
+
+## ${docGenConfiguration}
+
+## ${docJenkinsPluginDependencies}
+
+## Examples
+
+### AUnit test run via direct ABAP endpoint configuration in Jenkinsfile
+
+This  example triggers an AUnit test run directly on the ABAP endpoint.
+
+In order to trigger the AUnit test run you have to pass the username and password for authentication to the ABAP endpoint via parameters as well as the ABAP endpoint/host. You can store the credentials in Jenkins and use the abapCredentialsId parameter to authenticate to the ABAP endpoint/host.
+
+This must be configured as following:
+
+```groovy
+abapEnvironmentRunAUnitTest(
+    abapCredentialsId: 'abapCredentialsId',
+    host: 'https://myABAPendpoint.com',
+    aUnitConfig: 'aUnitConfig.yml',
+    script: this,
+)
+```
+
+To trigger the AUnit test run an AUnit config file `aUnitConfig.yml` will be needed. Check the section 'AUnit config file example' for more information.
+
+### AUnit test run via Cloud Foundry Service Key example in Jenkinsfile
+
+The following example triggers an AUnit test run via reading the Service Key of an ABAP instance in Cloud Foundry.
```

Comment:
We need to be consistent with the capitalisation of "Service Key" / "service key". I would tend to write it in lower case, but would also accept upper case. But it needs to be consistent